validator/client/registration.go

```diff
@@ -74,7 +73,6 @@ func (v *validator) SignValidatorRegistrationRequest(ctx context.Context, signer
 	} else {
 		sig, err := signValidatorRegistration(ctx, signer, newValidatorRegistration)
 		if err != nil {
-			log.WithError(err).Error("failed to sign builder validator registration obj")
```

Comment:
More of a caller's responsibility to log this. Logging it on the caller side now